experimental/dispatch_profiler/generator.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+
+from library import *
+from matmul import *
+from manifest import *
```

Comment:
It's recommended to import modules instead classes in the style guide (https://google.github.io/styleguide/pyguide.html#22-imports). So probably something like:
```suggestion
import library
import matmul
import manifest
```
And references the classes with full names below, like `manifest.Manifest`.

Reply:
I have thought about this style comment and tried it out, but this style increases the redundency in this code-structure. For example,

```
--- a/experimental/dispatch_profiler/generator.py
+++ b/experimental/dispatch_profiler/generator.py
@@ -2,7 +2,8 @@ import argparse
 
 from library import *
 from matmul import *
-from manifest import *
+#from manifest import *
+import manifest
 
 ###############################################################################
 
@@ -27,7 +28,7 @@ if __name__ == "__main__":
   args = parser.parse_args()
 
   # Manifests dispatches for a group of accompanying operations and configurations.
-  manifest = Manifest(args)
+  manifest = manifest.Manifest(args)
```

`+  manifest = manifest.Manifest(args)` needs to many strings reading "manifest". As of now the filenames/packages only structured to contain single functionality/separation of concerns so we can keep import as is, if things grow and we have files as proxy to namespaces, where a file contains glossary of functionality as classes, we can switch to the suggested style of imports.